internal/services/containerapps/container_app_environment_resource.go

```diff
@@ -310,6 +324,10 @@ func (r ContainerAppEnvironmentResource) Read() sdk.ResourceFunc {
 						state.PlatformReservedDnsIP = pointer.From(vnet.PlatformReservedDnsIP)
 					}
 
+					if appLogsConfig := props.AppLogsConfiguration; appLogsConfig != nil {
+						state.LogsDestination = *appLogsConfig.Destination
+					}
```

Comment:
if an existing Container App Environment (which doesn't set `LogsDestination`) is imported into Terraform/queried via the API - is the `Destination` field null/have a default value?

Reply:
Depends on the existing setting. A default creation, without specifying a LAW has the LogsDestination as an empty object in the JSON.